pkg/cli/scan.go

```diff
@@ -60,10 +59,12 @@ func Scan() *cobra.Command {
 
 type scanParams struct {
 	requireZeroFindings bool
+	localDBFile         string
 }
 
 func (p *scanParams) addFlagsTo(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&p.requireZeroFindings, "require-zero", false, "exit 1 if any vulnerabilities are found")
+	cmd.Flags().StringVar(&p.localDBFile, "import-local-grype-db", "", "import a local grype db")
```

Comment:
nit: I'd try to express in the flag name or even just the description text that what we're asking for here is a local filesystem path to the Grype DB